functionnal_tests/cypress/integration/login/username_popup_spec.js

```diff
@@ -149,6 +149,7 @@ describe('Login', function () {
         .click()
     })
     it('should not open the set username popup', function () {
+      this.skip() // FIXME - GB - 2020-09-09 - this tests is unstable and it will be fixed at https://github.com/tracim/tracim/issues/3483
```

Comment:
```suggestion
      this.skip() // FIXME - GB - 2020-09-09 - this test is unstable and it will be fixed at https://github.com/tracim/tracim/issues/3483
```